src/components/tabs/sell.tab.tsx

```diff
@@ -33,15 +37,78 @@ export function useSellTab(): StyledTabProps {
   };
 }
 
+interface ServicesContentProps {
+  selectedAsset?: Asset;
+  sellableAssets?: Asset[];
+}
+
+function ServicesContent({ selectedAsset, sellableAssets }: ServicesContentProps): JSX.Element {
+  const [step, setStep] = useState<SellTabStep>();
+  const { authenticationToken } = useAuthContext();
+  const { sync } = useSessionContext();
+
+  useEffect(() => {
+    if (authenticationToken) {
+      if (step !== SellTabStep.SELL_PROCESS) setStep(SellTabStep.SELL_PROCESS);
+    } else {
+      if (step !== SellTabStep.OVERVIEW) setStep(SellTabStep.OVERVIEW);
+    }
+  }, [authenticationToken]);
```

Comment:
Does this work with WalletConnect that we directly login if there is an authentication token?

Reply:
Yes, based on my tests it works well for login. However, I've not tested completing a transaction to see whether the connection to the wallet can be reestablished, so we still have to do that.